frontend/src/__mocks__/mockProjectK8sResource.ts

```diff
@@ -0,0 +1,39 @@
+import { ProjectKind } from '~/k8sTypes';
+
+type mockResourceConfigType = {
```

Comment:
I think we need a linter for this -- https://github.com/typescript-eslint/typescript-eslint/pull/1318 seems it's part of the typescript config.

Reply:
i renamed them all for now